code/modules/mob/dead/observer/observer.dm

```diff
@@ -657,6 +657,9 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		return TRUE
 
 	return FALSE
+	
+/mob/dead/observer/incapacitated()
+	return TRUE

```

Comment:
Yea, admin interaction with topics uses CanUseTopic(), which uses the nano interaction states. These have a separate checks for living mobs and observers.